docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+dataset contains the label "sky" which is not found in the `COCO <https://cocodataset.org/#home>`_
+dataset. As such, it may be necessary to look through the class labels supported
+by the model. You can do so with the following command:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface detect-ids  -\-model_type 'facebook/maskformer-swin-tiny-coco'` \
+      | The detect ID-to-label mapping for \`facebook/maskformer-swin-tiny-coco\` can be found at https://huggingface.co/facebook/maskformer-swin-tiny-coco/blob/main/config.json under the \`id2label\` key.
+
+Pipeline
+^^^^^^^^
+
+The task of blurring computer related objects can be achieved with the following
+``pipeline_config.yml``:
+
+   .. code-block:: yaml
+      :linenos:
+   
+      nodes:
+      - input.visual:
+          source: /path/to/cat_and_computer.mp4
+      - model.huggingface_hub:
+          task: instance_segmentation
+          model_type: facebook/maskformer-swin-tiny-coco
+          detect: ["laptop", "keyboard", "mouse"]
+      - draw.instance_mask:
+          effect:
+            blur: 50
+      - output.screen
+
+Here is a step-by-step explanation of what has been done:
+
+   | Line 2 :mod:`input.visual` tells PeekingDuck to load ``cat_and_computer.mp4``.
+   | Line 4 :mod:`model.huggingface_hub` is set to perform the ``instance_segmentation``
+   |        task and load the ``facebook/maskformer-swin-tiny-coco`` model. We also
+   |        set the model to detect only laptops, keyboards, and mice.
+   | Line 8 :mod:`draw.instance_mask` is used to visualize the output of the
+   |        Hugging Face Hub model. A blur effect is applied on the instance mask
+   |        outputs.
+
+Run the above with the command :greenbox:`peekingduck run`. 
+
+   .. figure:: /assets/tutorials/ss_huggingface_instance_segmentation.png
+      :width: 416
+      :alt: Cat and Computer Screenshot with Computer Related Objects Blurred
+   
+      Cat and Computer Screenshot with Computer Related Objects Blurred
+
+
+You should see a display of the ``cat_and_computer.mp4`` with the computer related
+objects blurred. The 30-second video will auto-close at the end, or you can press
+:greenbox:`q` to end early.
+
+
+Object Detection
+----------------
+
+This example also demonstrates how to blur computer related objects but with Hugging
+Face Hub's object detection models instead.
+
+Supported Models
+^^^^^^^^^^^^^^^^
+
+Use the following command to get a list of supported object detection models:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface models -\-task 'object detection'` \
+      | Supported Hugging Face \`object_detection\` models:
+      | facebook/detr-resnet-50
+      | facebook/detr-resnet-50-dc5
+      | hustvl/yolos-base
+      | hustvl/yolos-small
+      |
+      | <omitted>
```

Comment:
Suggest rephrasing `<omitted>` as `<long list truncated>`, as `<omitted>` alone might be misconstrued as part of the output.

Reply:
Changed